storage/src/mmr/journaled.rs

```diff
@@ -85,7 +85,7 @@ impl<B: Blob, E: RStorage<B>, H: Hasher> Mmr<B, E, H> {
             let item = journal.read(i).await?;
             vec.push(item);
         }
-        let mem_mmr = MemMmr::init(vec, oldest_peak);
+        let mem_mmr = MemMmr::init(vec, oldest_peak, vec![]);
```

Comment:
It seems like there are indeed old peaks we want to provide here (not empty `vec![]`)?

Reply:
We haven't implemented forgetting in the journaled MMR yet, so peaks won't ever be "old".

Reply:
oh how weird, seems I managed to edit your comment with my reply.

Reply:
changed to bootstrap in the "forget_all" state (in which case there are old peaks).